### PR TITLE
Add doc for one time system setup

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: custom_dc/setup_gcp.md [hunk 1]
```diff
@@ -3,5 +3,73 @@ layout: default
...
+
+## Overview
+
+Custom Data Commons is deployed in Google Cloud Platform (GCP). The follow
+manual presents how to install a custom Data Commons instance in an existing GCP
+project (with id `PROJECT_ID`).
+
+### Domain
+
+The installation can use an existing domain, or create a default domain
```
Comment (Fructokinase) [inline custom_dc/setup_gcp.md:17]: nit: Add a note saying that this guide uses the default domain? Maybe contact DC team for how to set up custom with custom domains? 

custom domains are supported after this PR.
https://github.com/datacommonsorg/website/pull/2218

File: custom_dc/setup_gcp.md [hunk 2]
```diff
@@ -3,5 +3,73 @@ layout: default
...
+   ![fa](/assets/images/custom_dc/install_step_2.png){: width="600" }
+
+   Note: If this step fails, please contact support+custom@datacommons.org with
+   the errors.
+
+1. Please run the following installation command inside the terminal. This may
+   take up to 20 minutes to complete.
+
+   ```bash
+   curl -fsSL https://raw.githubusercontent.com/datacommonsorg/website/master/scripts/install_custom_dc.sh -o install_custom_dc.sh && \
```
Comment (Fructokinase) [inline custom_dc/setup_gcp.md:44]: Can you help me change this to https://raw.githubusercontent.com/datacommonsorg/website/custom-dc-v0.1.0/scripts/install_custom_dc.sh please? Thanks!

**Review by Fructokinase — APPROVED** [review]
Thanks for doing this!!

File: custom_dc/setup_gcp.md (hunk 1 above)
Comment (shifucun) [reply custom_dc/setup_gcp.md:17]: Added note. Can you file a bug to add instructions on how to use an existing domain.

File: custom_dc/setup_gcp.md (hunk 2 above)
Comment (shifucun) [reply custom_dc/setup_gcp.md]: Done

**Pull request merged**